tests/test_acasclient.py

```diff
@@ -1797,4 +1770,296 @@ def test_044_author_and_role_apis(self):
+        # Test for csv format file
+        data_file_to_upload = Path(__file__).resolve()\
+            .parent.joinpath('test_acasclient', 'uniform-commas-with-quoted-text.csv')
+        self.experiment_load_test(data_file_to_upload, True)
+        self.experiment_load_test(data_file_to_upload, False)
+        txt_file = csv_to_txt(data_file_to_upload, self.tempdir)
+        self.experiment_load_test(txt_file, True)
+        self.experiment_load_test(txt_file, False)
+
+    @requires_basic_cmpd_reg_load
+    def test_non_unitform_comma_csv(self):
+        # Test for non-uniform comma format file
+        data_file_to_upload = Path(__file__).resolve()\
+            .parent.joinpath('test_acasclient', 'non-uniform-commas-with-quoted-text.csv')
+        self.experiment_load_test(data_file_to_upload, True)
+        self.experiment_load_test(data_file_to_upload, False)
+        txt_file = csv_to_txt(data_file_to_upload, self.tempdir)
+        self.experiment_load_test(txt_file, True)
+        self.experiment_load_test(txt_file, False)
+
+    @requires_basic_cmpd_reg_load
+    def test_malformed_single_quote(self):
+
+        data_file_to_upload = Path(__file__).resolve()\
+            .parent.joinpath('test_acasclient', 'malformatted-single-quote.csv')
+        response = self.experiment_load_test(data_file_to_upload, True)
+        self.assert_malformed_single_quote_file(response)
+        txt_file = csv_to_txt(data_file_to_upload, self.tempdir)
+        response = self.experiment_load_test(txt_file, True)
+        self.assert_malformed_single_quote_file(response)
+
+    @requires_basic_cmpd_reg_load
+    def test_speed(self):
+        # Speed test dry run
+        try:
+            # Dry run on 50 K row file with 3 columns of data should take
+            # less than 25 seconds to complete. On my machine it takes
+            # about 9 seconds. This is a sanity check to make sure the
+            # dry run hasn't slowed significantly.
+            with Timeout(seconds=25):
+                data_file_to_upload = Path(__file__).resolve()\
+                    .parent.joinpath('test_acasclient', '50k-lines.csv')
+                self.experiment_load_test(data_file_to_upload, True)
+        except TimeoutError:
+            self.fail("Timeout error")
+        else:
+            pass
+
+    @requires_basic_cmpd_reg_load
+    def test_experiment_loader_curve_validation(self):
+        # Test dose response curve validation
+        data_file_to_upload = Path(__file__).resolve()\
+            .parent.joinpath('test_acasclient', '4 parameter D-R-validation.csv')
+
+        response = self.experiment_load_test(data_file_to_upload, True)
+
+        # When loading Dose Resposne format but not having ACAS fit the curves, we shold get a dose response summary table
+        self.assertTrue(response['results']['htmlSummary'].find("bv_doseResponseSummaryTable") != -1)
+
+        # print(json.dumps(response['errorMessages'], sort_keys=True, indent=4))
```

Comment:
To be removed?

Reply:
I left that there on purpose in case we need to update the expected results dict.  I added a new comment to clarify why the comment was left in the code.